pkg/web/templates/components/transaction_rejection.html

```diff
@@ -0,0 +1,46 @@
+<dialog id="transaction_rejection_modal" class="modal">
```

Comment:
The transaction rejection modal was moved so that it may be reused.

Reply:
nice!